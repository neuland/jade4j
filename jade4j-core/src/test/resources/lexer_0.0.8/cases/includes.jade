
include auxiliary/mixins

+foo

body
  include auxiliary/smile.html
  include auxiliary/escapes.html
  script(type="text/javascript")
    include:js auxiliary/includable.js
