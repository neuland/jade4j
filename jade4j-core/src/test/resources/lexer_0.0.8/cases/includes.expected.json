{"type":"newline","line":2}
{"type":"include","line":2,"val":"auxiliary/mixins"}
{"type":"newline","line":4}
{"type":"call","line":4,"val":"foo"}
{"type":"newline","line":6}
{"type":"tag","line":6,"val":"body","selfClosing":false}
{"type":"indent","line":7,"val":2}
{"type":"include","line":7,"val":"auxiliary/smile.html"}
{"type":"newline","line":8}
{"type":"include","line":8,"val":"auxiliary/escapes.html"}
{"type":"newline","line":9}
{"type":"tag","line":9,"val":"script","selfClosing":false}
{"type":"attrs","line":9,"attrs":[{"name":"type","val":"\"text/javascript\"","escaped":true}]}
{"type":"indent","line":10,"val":4}
{"type":"include","line":10,"val":"auxiliary/includable.js","filter":"js","attrs":null}
{"type":"outdent","line":11}
{"type":"outdent","line":11}
{"type":"eos","line":11}